reference-architecture/aws-ansible/playbooks/roles/rhsm-repos/tasks/main.yaml

```diff
@@ -1,11 +1,12 @@
 ---
 - name: disable unneeded repos
   command: subscription-manager repos --disable='*'
-  when: register_result | changed
+  when: register_result | changed and ansible_distribution == "RedHat"
```

Comment:
I have a feeling this does not works...  I used different condition but got failure:
`fatal: [ose-master01.bgol.lt]: FAILED! => {"failed": true, "msg": "The conditional check 'register_result | changed and deployment_type != \"origin\"' failed. The error was: |changed expects a dictionary\n\nThe error appears to have been in '/home/mangis/workspace/openshift-ansible-contrib/reference-architecture/aws-ansible/playbooks/roles/rhsm-repos/tasks/main.yaml': line 2, column 3, but may\nbe elsewhere in the file depending on the exact syntax problem.\n\nThe offending line appears to be:\n\n---\n- name: disable unneeded repos\n  ^ here\n"}`